ulog/README.md

```diff
@@ -1,16 +1,15 @@
 # Logging package
 
 `package ulog` provides an API wrapper around the logging library
-[zap](https://github.com/uber-go/zap). `package ulog` uses the builder pattern
-to instantiate the logger. With `LogBuilder` you can perform pre-initialization
+[zap](https://github.com/uber-go/zap). `package ulog` uses zap's builder
+to instantiate the logger. With zap's `Builder` you can perform pre-initialization
```

Comment:
There is no `Builder` type, and there's no longer any support for injecting custom loggers.